README.md

```diff
@@ -47,6 +47,10 @@ Read more about the package, and the intent behind it, in the [announcement on s
 - [`split(maxSplits:omittingEmptySubsequences:whereSeparator)`, `split(separator:maxSplits:omittingEmptySubsequences)`](https://github.com/apple/swift-algorithms/blob/main/Guides/LazySplit.md): Lazy versions of the Standard Library's eager operations that split sequences and collections into subsequences separated by the specified separator element.
 - [`windows(ofCount:)`](https://github.com/apple/swift-algorithms/blob/main/Guides/Windows.md): Breaks a collection into overlapping subsequences where elements are slices from the original collection.
 
+#### Binary Search Partitions
```

Comment:
If this deserves its own heading, then it should follow the same capitalization and grammatical phrasing as other headings, and it should be above "Other useful operations"; otherwise, it should be added to the latter section in alphabetical order.

Reply:
Thanks for the feedback @xwu! I was following issue #131 " It needs to be in a different category, though."

Reply:
Let's add it to "Other useful operations" for now, thanks!